### ISSUE TITLE
Currently some issues with making the Nagios config

### ISSUE THREAD

**Issue by 153957** [HiSPARC/publicdb#126]
Probably related to some badly configured service in the database;

```python
 ...
File "../django_publicdb/inforecords/views.py", line 83, in create_nagios_config
   check_command += ('!%s:%s!%s:%s' % vars)
TypeError: not enough arguments for format string
```

raised when tietar requests the new config.

**Comment by davidfokkema** [comment]
When did this start?

**Comment by 153957** [comment]
I got the Internal Server error emails yesterday after I modified the contact email for the Science Park stations (i.e. all the ones that used to email me) to beheer@hisparc.nl, which triggered tietar to try to reload the config. I also changed the Cluster for some of those stations (the test stations in the 9# range) to Amsterdam (required because now the station number must be in the same 'hundreds' as the cluster it is in).
